File: DheerajKumar/problem10.cpp
//10. WAP to check whether the number is positive, negative or zero.

#include<iostream>
using namespace std;

bool checkNegative(int number){
    if(number < 0){
        return true;
    }
    return false;
}

bool checkPositive(int number){
    if(number > 0){
        return true;
    }
    return false;
}

int main(){

    int number;
    cout<<"Enter number ->>";
    cin>>number;

    if(number == 0){
        cout<<" The number is ZERO";
        return 0;
    }
    if(checkPositive(number)){
        cout<<"The number is POSITIVE";
        return 1;
    }
    if(checkNegative(number)){
        cout<<" The number is NEGATIVE";
        return -1;
    }
}

File: LovelyKumari/BasicProgramming/Problem10.java
import java.util.Scanner;

public class Problem10 {
    // WAP to check whether the number is positive, negative or zero
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a number : ");
        int num=sc.nextInt() ;

        if(num > 0){
            System.out.println(num + " is a positive number.");
        } else if(num < 0){
            System.out.println( num + num + " is a negative number.");
        } else {
            System.out.println("Number is zero ");
        }
        
    }

    
}
